pkg/agent/openflow/client.go

```diff
@@ -181,19 +181,16 @@ func (c *client) InstallClusterServiceCIDRFlows(serviceNet *net.IPNet, gatewayOF
 }
 
 func (c *client) InstallGatewayFlows(gatewayAddr net.IP, gatewayMAC net.HardwareAddr, gatewayOFPort uint32) error {
```

Comment:
Questions:
1. Why not change other funcs, like Pod and Node flows?
2. Do we still need addMissingFlows?
@antoninbas

Reply:
We can use bundle for the other Install* methods a well.
If we use bundle for InstallPodFlows and InstallNodeFlows, I do not believe we need `addMissingFlows` any more if we can guarantee that these functions are only called once for a given Pod / Node.

Reply:
@wenyingd I'm currently using the flow cache for flow reconciliation when there is a reconnection, so I do think we need to keep it unless we decide to do the reconciliation some other way. Don't we also need it for flow deletion anyway?

Is there any drawback to using bundle for any operation which involves at least 2 flow operations? If not, I would use it everywhere, even for deletion.

Reply:
There is no drawback to use bundle for add/modify/delete operation. I want to confirm it is because I need more enhancement in ofnet(binding) to support all these operations.